content/millennium/r4/base/entities/location.md

```diff
@@ -60,24 +60,26 @@ Search for Locations that meet supplied query parameters:
 
  Name                 | Required?               | Type         | Description
 ----------------------|-------------------------|--------------|-------------------------------------------------------
- `_id`                | This or `-physicalType` | [`token`]    | The logical resource id associated with the resource.
- `-physicalType`      | This or `_id`           | [`token`]    | The location’s physical type. Example: `http://terminology.hl7.org/CodeSystem/location-physical-type|ro`
- `identifier`         | no                      | [`token`]    | The location’s identifier. Example: `653385|FSI^~BUILD^~NU` 
- [`_count`]           | no                      | [`number`]   | The maximum number of results to return. Defaults to `100`.
- `address`            | no                      | [`string`]   | A (part of the) address of the location.
- `address-city`       | no                      | [`string`]   | A city specified in an address
- `address-state`      | no                      | [`string`]   | A state specified in an address
- `address-postalcode` | no                      | [`string`]   | A postal code specified in an address
- `name`               | no                      | [`string`]   | A portion of the location's name or alias
- `organization`       | no                      | [`reference`]| Searches for locations that are managed by the provided organization
+ `_id`                | Conditionally           | [`token`]    | The logical resource ID associated with the resource. This parameter is required if the `-physicalType` parameter is not used.
```

Comment:
@cldeitrich, this doesn't match what we've been doing. do we need to go back and update the resources we've previously done with this "Conditionally" plus the description updates?

Reply:
Summer and I are planning to use this format going forward as we think it's clearer than the "This, or" construction that was used previously. We are open to other suggestions as well. For the resources that were already completed, we can apply our edits when we migrate the doc over to Oracle -- but if you have time to update the resources before then, that works too. :)